cli/Makefile

```diff
@@ -56,5 +56,4 @@ setup:
 clean:
 	rm -rf build/ semgrep.egg-info/ .eggs/
 	rm -rf .pytest_cache/ .benchmarks/
-	rm -f semgrep/bin/semgrep-core
-	rm -rf semgrep/__pycache__ semgrep/*/__pycache__
```

Comment:
I'm not sure what the purpose of this build target is, but keeping __pycache__ files around shouldn't really matter.